docs/rustdoc/centralized_telescope/main.md

```diff
@@ -1,3 +1,51 @@
 # Telescope - Construction with Bounded DFS
-This is the main page of centralized documentation.
-It will include a generic explanation of the protocol.
+
+In scenarios where $n_p$ is small, the parameters given for prehashed construction (relative to the security parameter, $\lambda$) are not optimal, resulting in a reduced probability of constructing a valid proof in a single attempt.
+For large $n_p$, the rapid growth in potential proof tuples ensures valid ones can be found efficiently, allowing the prehashed construction to work seamlessly.
+In contrast, small $n_p$ limits the search space, making the previous parameters inadequate.
+To address this, the **construction with bounded DFS** expands on the prehashed version with *retries*, *randomized indexing*, and *bounding* the DFS.
+
+## Overview
+- When $n_p$ is large, the rapid growth in potential proof tuples increases the chances of finding a valid proof, making construction easier.
+- For small $n_p$, limited elements reduce the probability of finding a valid proof in a single attempt.
```

Comment:
The real reason is that when n_p is small, the arrangement of balls into bins is far from uniform. On average each bin still gets exactly one ball, but it's likely that many bins will be empty and some will have many elements. Turns out it's better to have many bins with one ball than one bin with many balls.

Reply:
When n_p is small, we still can use the scheme with prehashing, but the completeness error is moderate instead of tiny. By introducing retries, the completeness error is decreased.